src/config/env.ts

```
@@ -0,0 +1,25 @@
+export const API_BASE_URL = process.env.DEV_SERVER_URL ?? 'http://localhost:8080';
+export const NEXT_BASE_URL = process.env.NEXT_PUBLIC_BASE_URL ?? 'http://localhost:3000';
+
+export const KAKAO_JS_KEY = process.env.NEXT_PUBLIC_JAVASCRIPT_KEY!;
+export const KAKAO_REST_API_KEY = process.env.REST_API_KEY!;
+export const KAKAO_CLIENT_SECRET = process.env.CLIENT_SECRET!;
+
+export const SUPABASE_URL = process.env.NEXT_PUBLIC_SUPABASE_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)